presto-main/src/test/java/com/facebook/presto/operator/OperatorAssertion.java

```diff
@@ -62,6 +63,21 @@ private OperatorAssertion()
 
     public static List<Page> toPages(Operator operator, Iterator<Page> input)
     {
+        try {
+            return toPagesUnsafe(operator, input);
+        }
+        finally {
+            try {
+                operator.close();
```

Comment:
ok, I have migrated all calls to `OperatorAssertions` to new methods, which takes `OperatorFactory` and `DriverContext`. See bunch of commits at the end of the PR